src/components/ebay-button/README.md

```diff
@@ -18,6 +18,7 @@ Name | Type | Stateful | Description
 `disabled` | Boolean | Yes |
 `partially-disabled` | Boolean | No
 `variant` | String | No | optional, to alter Skin classes: "expand" / "cta"
+`pill` | Boolean | No | for buttons/links that look like a pill (DS6 only)
```

Comment:
@msendlakowski In the `README.md` can you also add a simple example of the markup for the pill use case, along with the simple `<ebay-button>` tag?